itsy.css

```
@@ -0,0 +1,87 @@
+body {
+  margin: 0px;
+}
```

add spaces after bracket `}`
```suggestion
}

```